src/main/java/codeu/controller/ProfileServlet.java

```diff
@@ -0,0 +1,54 @@
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class ProfileServlet extends HttpServlet {
+    
+     private UserStore userStore;
+    
+    @Override
+    public void init() throws ServletException {
+        super.init();
+        setUserStore(UserStore.getInstance());
+    }
+    /*
+     * Get the profile page
+     */
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response)
```

Comment:
Looks good to me :)
You may need to add/change this when going through the JSP work & testing, but thats pretty normal